### ISSUE TITLE
Should person.shex limit the number of WebID's to just one?

### ISSUE THREAD

**Issue by pmcb55** [inrupt/pod-browser#251]
As pointed out by @matthieubosquet [here](https://github.com/inrupt/pod-browser/pull/246#discussion_r574068992), shouldn't a Person be able to maintain any number WebID's?

Currently the ShEx (in `shapes/contact/person.shex`) says:

```
vcard:Individual {
    vcard:fn xsd:string {1} ;

    # vcard:WebId is a custom term
    vcard:url vcard:WebId {1};
:
:
```

**Comment by megoth** [comment]
Ah, yes, I think that should be changed to + (meaning 1 or more).

**Comment by megoth** [comment]
Pinging @timbl who might want to weigh in on this

**Comment by matthieubosquet** [comment]
Ideally `*` that is 0 to many.

Not everyone is going to have a WebID from the get go and it would be a shame to be able to import close to 0 of your contacts because of that. I'm not sure what the point of this restriction is?

**Comment by megoth** [comment]
> Ideally `*` that is 0 to many.
> 
> Not everyone is going to have a WebID from the get go and it would be a shame to be able to import close to 0 of your contacts because of that. I'm not sure what the point of this restriction is?

You're correct, for an address book in general it doesn't make sense to limit it to one WebId or more. This was written in the context of PodBrowser, so it has been influenced because of that. In general I think it makes sense to have `*` yes, 0 to many.

I'm perfectly fine with changing it to `*`.
